packages/core/src/behavior.ts

```diff
@@ -229,26 +236,29 @@ export function createObservableBehavior<
     },
     receive: () => behavior,
     subscribe: (observer) => {
-      return observable.subscribe(observer);
+      return observable?.subscribe(observer);
```

Comment:
shouldnt `.subscribe` be always available on observables? 😄

Reply:
Don't worry, it is! This signifies that the _observable_ itself might not exist before `subscribe(...)` is called.

Not sure what to do here, it's an unlikely race condition that (in here) is handled by just not subscribing.